python/turbine_models/gen_external_params/gen_external_params.py

```diff
@@ -12,14 +12,17 @@
     default="meta-llama/Llama-2-7b-chat-hf",
 )
 parser.add_argument("--quantization", type=str, default="int4")
-parser.add_argument("--weight_path", type=str, default="Llama2_7b_i4quant.safetensors")
+parser.add_argument("--weight_path", type=str, default="Llama2_7b.safetensors")
```

Comment:
It's going to become increasingly hard to keep these straight. I would suggest deriving the default from the other parameters and including the precision in the default filename. Probably want to have it default to None and then handle that at the usage site.